src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss

```diff
@@ -6,7 +6,7 @@
   &:after {
     position: relative;
     content: "+";
-    top: 12px;
+    top: 4px;
     width: 20px;
     height: 20px;
     font-size: 16px;
```

Comment:
use the standard variable for font-size.

Reply:
done